src/main/python/sunburst_visualization/sunburst.py

```diff
@@ -0,0 +1,98 @@
+                                if metodus_elem["name"] == "":
+                                    character_.append(files["relativePath"]+("\\")+listaelem_class["name"]+("\\")+("None_method"))
+                                else:
+                                    character_.append(files["relativePath"]+("\\")+listaelem_class["name"]+("\\")+metodus_elem["name"])
+
+                                parent_.append(act_parent)
+                                values_.append(metodus_elem["och"])
+                                act_metodus_parent=character_[-1]
+
+                                for statement_elem in metodus_elem["statements"]:
+                                    character_.append(
+                                    files["relativePath"] + ("\\") + listaelem_class["name"] + ("\\") + metodus_elem[
+                                        "name"]+ ("\\") + statement_elem["line"])
+
+                                    parent_.append(act_metodus_parent)
+                                    values_.append(statement_elem["och"])
+
+                        else:
+                            character_.append(files["relativePath"]+("\\")+listaelem_class["name"])
+                            akt_class_parent = character_[-1]
+                            parent_.append(act_parent)
+                            values_.append(listaelem_class["och"])
+
+                            for metodus_elem in listaelem_class["methods"]:
+                                if metodus_elem["name"] == "":
+                                    character_.append(files["relativePath"]+("\\")+listaelem_class["name"]+("\\")+("None_method"))
+                                else:
+                                    character_.append(files["relativePath"]+("\\")+listaelem_class["name"]+("\\")+metodus_elem["name"])
+
+                                parent_.append(akt_class_parent)
+                                values_.append(metodus_elem["och"])
+                                act_metodus_parent = character_[-1]
+
+
+                                for statement_elem in metodus_elem["statements"]:
+                                    character_.append(
+                                files["relativePath"] + ("\\") + listaelem_class["name"] + ("\\") + metodus_elem[
+                                    "name"] + ("\\") + statement_elem["line"])
+
+                                    parent_.append(act_metodus_parent)
+                                    values_.append(statement_elem["och"])
+
+        data = dict(
+            character=character_,
+            parent=parent_,
+            value=values_
+            )
+
+        #[print(v) for k,v in data.items()]
+
+        fig = px.sunburst(
+            data,
+            names='character',
+            parents='parent',
+            values='value',
+            )
+#fig.show()
+        fig.write_html("sunburst.html")
+
+        sys.exit()
```

Comment:
i don't think we need the sys.exit(). 

Do we make anything in python after that? 
Still I think it is unnecessary,